presto-main/src/test/java/com/facebook/presto/operator/BenchmarkHashBuildAndJoinOperators.java

```diff
@@ -294,9 +296,11 @@ private LookupSourceFactory benchmarkBuildHash(BuildContext buildContext, List<I
         }
         operator.finish();
 
-        if (!hashBuilderOperatorFactory.getLookupSourceFactory().createLookupSource().isDone()) {
+        ListenableFuture<LookupSourceProvider> lookupSourceProvider = hashBuilderOperatorFactory.getLookupSourceFactory().createLookupSourceProvider();
+        if (!lookupSourceProvider.isDone()) {
             throw new AssertionError("Expected lookup source to be done");
```

Comment:
change "Expected lookup source to be done" to "Expected lookup source provider to be ready"